projects/admin/src/app/record/detail-view/template-detail-view/template-detail-view.component.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { RecordService } from '@rero/ng-core';
+import { Observable, of, Subscription } from 'rxjs';
+
+@Component({
+  selector: 'admin-template-detail-view',
+  templateUrl: './template-detail-view.component.html'
+})
+export class TemplateDetailViewComponent implements OnInit, OnDestroy {
+
+  /** Observable resolving record data */
+  record$: Observable<any>;
+
+  /** Observable of the imported record in marc format */
+  marc$: Observable<any>;
+
+  /** Record subscription */
+  private _recordObs: Subscription;
+
+  /** Resource type */
+  type: string;
+
+  /** Document record */
+  record: any;
+
+  /** Css class for dd in template */
+  ddCssClass = 'col-sm-9 mb-0';
+
+  /** Css class for dt in template */
+  dtCssClass = 'col-sm-3';
+
+  /** constructor
+   * @param _router - ActivatedRoute to get url parameters.
+   * @param _recordService - RecordService to the MARC version for the record.
+   */
+  constructor(
+    private _router: ActivatedRoute,
+    private _recordService: RecordService
+  ) { }
+
+  /** On init hook */
+  ngOnInit() {
+    this._recordObs = this.record$.subscribe((record: any) => {
```

Comment:
The suffix `Obs` is not correct. It is a subscription, not an observable.

Reply:
Done